doc: Document new/improved proxy and trust store options for Gatling JS (#79)
gatling/gatling.io-doc: content/reference/integrations/build-tools/js-cli.md

2. Run `npm install`

### Internet access

The `gatling` CLI tool will require Internet access to:

- Automatically download the Gatling runtime bundle, required to execute your Gatling simulations (from
  `https://github.com/gatling/gatling-js/releases/`).

>   - By default, it will be installed in the folder `~/.gatling` (Linux/macOS) or `%USERPROFILE%\.gatling` (Windows). You

Reviewer: ```suggestion
  - By default, it is installed in the folder `~/.gatling` (Linux/macOS) or `%USERPROFILE%\.gatling` (Windows). You
```
